src/drivers/sqs/Command.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\queue\sqs;
+
+use yii\queue\cli\Command as CliCommand;
+
+/**
+ * Manages application gearman-queue.
```

Comment:
This class is full copy of https://github.com/yiisoft/yii2-queue/blob/master/src/drivers/gearman/Command.php